app/views/accounts/_emails.html.slim

```diff
@@ -12,3 +12,7 @@
         = render @view_model.manage_action_partial,
           path: manage_email_url(id: email.id),
           name: t('account.index.email')
+        - if current_user.email_addresses.count > 1
```

Comment:
How would you feel about creating a policy object to handle this? Something like `EmailPolicy`. The `MfaPolicy` object has a pattern for that and an `EmailContext` decorator already exists.

Reply:
Will do